index.d.ts

```diff
@@ -129,6 +129,18 @@ export interface AssertContext {
 	 * Assert that function throws an error or promise rejects.
 	 * @param error Can be a constructor, regex, error message or validation function.
 	 */
+	 /**
+ 	 * Assert that value is deep equal to expected. (DEPRECATED, use `deepEqual`)
+ 	 */
+ 	same<U>(value: U, expected: U, message?: string): void;
+ 	/**
+ 	 * Assert that value is not deep equal to expected. (DEPRECATED use `notDeepEqual`)
```

Comment:
Move the `DEPRECATED` warning to the start?

IDE's use these comments for tooltips. So it would be good to have the warning as prominent as possible.